internal/pkg/containers/image/image.go

```diff
@@ -20,9 +21,15 @@ import (
 func Pull(ctx context.Context, reg config.Registries, client *containerd.Client, ref string) (img containerd.Image, err error) {
 	resolver := NewResolver(reg)
 
-	err = retry.Exponential(1*time.Minute, retry.WithUnits(1*time.Second)).Retry(func() error {
+	err = retry.Exponential(20*time.Minute, retry.WithUnits(5*time.Second), retry.WithErrorLog(true)).Retry(func() error {
```

Comment:
nit: magic numbers

Reply:
any suggestions?  would you like to see it configurable? expose at constants?